### PR TITLE
Added MIT License

### PR DESCRIPTION
**Pull request by lydia-at-amazon**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 UAVCAN
```
Comment (thirtytwobits) [inline LICENSE]:
```suggestion
Copyright (c) 2022 OpenCyphal
Copyright (c) 2022 Amazon.com
```

**Review by thirtytwobits — CHANGES_REQUESTED** [review]
Are there other licenses that are inherited from the vscode extension SDK? If so please note their copyright and licenses here.

File: LICENSE (hunk 1 above)
Comment (lydia-at-amazon) [reply LICENSE]: oops, thank you! Copy pasta

**Comment by lydia-at-amazon** [comment]
> 

Checking for other licenses

**Comment by lydia-at-amazon** [comment]
Don't think there are any other Licenses we need to add

**Review by thirtytwobits — APPROVED** [review]

**Pull request merged**